src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -337,6 +382,34 @@ abstract class AbstractCheckTask(
         }
     }
 
+    private fun doOnCompleteEvent(previousState: State) {
+        completeEvent(previousState)
```

Comment:
Why do we call `completeEvent` here? It should be called regardless of the  `untrusted` flag

Reply:
Refer to https://github.com/th2-net/th2-check1/pull/36#issuecomment-899487957

Reply:
Refer to https://github.com/th2-net/th2-check1/pull/36#issuecomment-899487957